src/gdclasses/OpenXRHand.cpp

```diff
@@ -139,8 +144,12 @@ void OpenXRHand::_physics_process(float delta) {
 
 			if (joints[i] != NULL) {
 				if (parents[i] == -1) {
-					// apply our reference frame to our root frame
-					t = reference_frame * t;
+					if (relative_transform) {
+						t = inv_transforms[XR_HAND_JOINT_PALM_EXT] * t;
```

Comment:
skipping the reference_frame will give you problems if you've ever called `center_on_hmd`

Reply:
Ah, you are right, I will add it back in and test it